### ISSUE TITLE
Move Operate `ExportHandlers` from feature branch to main

### ISSUE THREAD

**Issue by sdorokhova** [camunda/camunda#22642]
### Description

We implemented `ExportHandlers` on feature branch [feat-operate-exporter](https://github.com/camunda/camunda/tree/feat-operate-exporter/zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/handlers) but now we need to merge them to `main`. 

### TODO

- [x] move DecisionHandler: https://github.com/camunda/camunda/pull/23439
- [x] move VariableHandler: https://github.com/camunda/camunda/pull/23498
- [x] move IncidentHandler: https://github.com/camunda/camunda/pull/23562
- [x] move process instance handler: https://github.com/camunda/camunda/pull/23470
- [x] move SequenceFlowHandler: https://github.com/camunda/camunda/pull/23611
- [x] #23617
- [x] #23619
- [x] #23620
- [x] #23621
- [x] #23622
- [x] #23624
- [x] move PostImporterQueueFromIncidentHandler: https://github.com/camunda/camunda/pull/23601
- [x] #23618
- [ ] #23623
- [ ] #23616

### Acceptance Criteria

- [ ]  Zeebe exporter is able to export all necessary Operate data (business logic state will be at the time point when exporters were created in feature branch, we have [a separate issue](https://github.com/camunda/camunda/issues/22747) to "copy" all later updates from importer to exporter)
- [ ] Integration tests are refactored to use new exporter approach: minimal necessary amount must be green, others may be skipped for now when failing

**Comment by sdorokhova** [comment]
I move this task back in ready state, but I will pick it right after #19006  that I'm working on now.

**Comment by Zelldon** [comment]
I will close this issue, as:

* https://github.com/camunda/camunda/issues/23623 - will be covered by Tasklist task index
* https://github.com/camunda/camunda/issues/23616 - will be handled separately that is not part of the move of logic from branch to main
